src/main/zapHomeFiles/hudtutorial/en_GB/PageAlerts.html

```diff
@@ -8,15 +8,15 @@
 <div class="roundContainer">
 	<h1>Page Alerts</H1>
 	Potential security issues on the page you are viewing will be shown as 'Page Alerts', 
-	which by default are shown on the left hand side. This allows you to just focus on the potential issues on
+	which by default are shown on the left-hand side. This allows you to just focus on the potential issues on
 	your current page without being distracted by issues on other pages. 
 	<p>
-	Alerts have an associated risk, and all alerts with the same level of risk are shown together:
+	Each alert has an associated risk, and all alerts with the same level of risk are shown together:
 	<ul>
-	<li><img src="page-alerts-high.png"> High Level Page Alerts
-	<li><img src="page-alerts-medium.png"> Medium Level Page Alerts
-	<li><img src="page-alerts-low.png"> Low Level Page Alerts
-	<li><img src="page-alerts-informational.png"> Informational Level Page Alerts
+	<li><img src="page-alerts-high.png" alt="[red flag]"> High-Level Page Alerts
```

Comment:
These no longer match the labels shown in the dialogue, should those be changed?

Reply:
I'm not sure these ever matched:
https://github.com/zaproxy/zaproxy/blob/d91db825c988f2be808738983360fd31731815fd/zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties#L1656-L1661

I can't find anything else that corresponds to this...

Reply:
I meant in the HUD's UI:
https://github.com/zaproxy/zap-hud/blob/37c57279999707bdf3a644297d7a4e2ba25c8a70/src/other/resources/UIMessages/UIMessages.properties#L27

they are not the same either... thought they were, sorry.

Reply:
To be fair, the fact that these three places do not agree is probably a bug.

And you were right in guessing that I wasn't checking.